sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_v1/query.sql

```diff
@@ -46,20 +46,17 @@ store_stats AS (
   ON
     country_name = name
 ),
-new_profiles_and_activations AS (
+_new_profiles AS (
   SELECT
     first_seen_date AS `date`,
-    country,
-    SUM(new_profile) AS new_profiles,
-    SUM(activated) AS activations,
+    first_reported_country AS country,
+    COUNT(*) AS new_profiles,
   FROM
-    firefox_ios.new_profile_activation
+    firefox_ios.firefox_ios_clients
   WHERE
-    submission_date >= '2022-01-01'
+    DATE(submission_timestamp) >= '2022-01-01'
     AND first_seen_date >= '2022-01-01'
-     -- below filter required due to untrusted devices anomaly,
-     -- more information can be found in this bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1846554
-    AND NOT (app_display_version = '107.2' AND submission_date >= '2023-02-01')
+    -- TODO: do we need to filter here for "release" channel only?
```

Comment:
@bani should we be filtering here for `release` channel only?

Reply:
yes, the funnel should be release channel only